tests/lax_numpy_test.py

```diff
@@ -25,6 +25,7 @@
 import unittest
 from unittest import SkipTest
 import warnings
+from packaging.version import Version
```

Comment:
No need to use `packaging.version` here: we already define `numpy_version` below.